power_grid_model_io_native_c/power_grid_model_io_native_c/src/vnf_pgm_converter.cpp

```diff
@@ -0,0 +1,34 @@
+// SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#define PGM_IO_DLL_EXPORTS
+
+#include <power_grid_model_io_native/vnf_converter/vnf_pgm_converter.hpp>
+
+#include "handle.hpp"
+#include <power_grid_model_io_native_c/basics.h>
+#include <power_grid_model_io_native_c/vnf_pgm_converter.h>
+
+#include <power_grid_model/auxiliary/dataset.hpp>
+
+using power_grid_model::ConstDataset;
+
+PGM_IO_VnfConverter* PGM_VNF_create_converter(const PGM_IO_Handle* /*handle*/, char* file_buffer) {
```

Comment:
In this case, I believe we should leave it not const. There are multiple ways how it can be set within the converter.
1. It can be set by simply passing it to the converter
2. If nothing is passed then there is a default value for it, which will have to be overridden at some point in the conversion process.
Because of 2, I do not think we can make it const.